server/templates/home.html

```diff
@@ -8,83 +9,87 @@
     <div class="row">
       {% if rulesDocs != None %}
       <div class="col-lg-3 col-md-3 col-sm-4">
-        <div class="list-group table-of-contents">
-          {% for document in rulesDocs %}
-          <a class="list-group-item" href="{{document[1]}}"> {{document[0]}} </a>
-          {% endfor %}
-        </div>
+
+        {{ list.holoListOfLinks(rulesDocs) }}
+
       </div>
       {% endif %}
 
       {% if character_docs != None %}
       <div class="col-lg-3 col-md-3 col-sm-4">
-        <div class="list-group table-of-contents">
-          {% for document in character_docs %}
-          <a class="list-group-item" href="{{document[1]}}"> {{document[0]}} </a>
-          {% endfor %}
-        </div>
+
+        {{ list.holoListOfLinks(character_docs) }}
+
       </div>
       {% endif %}
 
       {% if itemsDocs != None %}
       <div class="col-lg-3 col-md-3 col-sm-4">
-        <div class="list-group table-of-contents">
-          {% for document in itemsDocs %}
-          <a class="list-group-item" href="{{document[1]}}"> {{document[0]}} </a>
-          {% endfor %}
-        </div>
+
+        {{ list.holoListOfLinks(itemsDocs) }}
+
       </div>
       {% endif %}
 
       <div class="col-lg-3 col-md-3 col-sm-4">
-        <div class="list-group table-of-contents">
-          <a class="list-group-item" href="npcgen">NPC Generator</a>
-          <a class="list-group-item" href="files"> Files </a>
-          <a class="list-group-item" href="gamelogs"> Game Logs </a>
-          <a class="list-group-item" href="missions">Missions</a>
-          <a class="list-group-item" href="missionfiles">Mission Files</a>
-        </div>
+        {% set miscLinks = [
+          ['NPC Generator', 'npcgen'],
+          ['Files', 'files'],
+          ['Game Logs', 'gamelogs'],
+          ['Missions', 'missions'],
+          ['Mission Files', 'missionfiles']
+        ]%}
+
+        {{ list.holoListOfLinks(miscLinks) }}
+
       </div>
 
       {% if 'username' in session %}
-      <!-- TODO: make only available to admins -->
+      {# TODO: make only available to admins #}
```

Comment:
I shouldn't even have TODO's anywhere in my code... I wrote this before I read Clean Code...